### PR TITLE
First attempt in mocking patch function

### PR DESCRIPTION
**Pull request by x24ling**
#26

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Release Notes
```
Comment (dkarchmer) [inline RELEASE.md]: Use CHSANGELOG.md file instead

File: iotile_cloud/utils/mock_cloud.py [hunk 2]
```diff
@@ -263,15 +263,24 @@ def _get_streamer_ack(self, device_id, index):
...
+        if(request.method == 'PATCH'):
+            payload = json.loads(request.data)
+            for key in payload.keys():
+                if key not in container[obj_id].keys():
+                    raise ErrorCode(400)
+            for key,value in payload.iteritems():
+                container[obj_id][key] = value
+            return container[obj_id]
+        else: #Assuming method is GET instead
+            pass
```
Comment (dkarchmer) [inline iotile_cloud/utils/mock_cloud.py]: You need to return something like before for GET:

```
else:
   return container[obj_id]
```

**Review by dkarchmer — CHANGES_REQUESTED** [review]

**Review by dkarchmer — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+### 0.8.4
```
Comment (timburke) [inline CHANGELOG.md:1]: Style match, all of the other entries are `vX.Y.Z (DATE)`

**Review by timburke — APPROVED** [review]
See small tweak on release notes

**Pull request merged**